starter_code/style/style.css

```
@@ -0,0 +1,137 @@
+
+body{
+    font-family: 'Bitter', serif;
+}
+header{
+    background-color: black;
+    height: 100px;
+    margin-top:0px;
+    border-bottom: 10px;
+    border-color: yellow;
+    border-style: solid;
+}
+h1{
+    color: yellow;
+    font-size: 45px;
+    padding:10px;
+}
+.nav{
+    float: right;
+    margin-right: 7px;
+}
+.nav li{
+    display: inline;
+    color: white;
+    margin: 10px 6px;
+}
+a{
+    text-decoration: none;
+    color: white;
+}
+a:hover{
+    color: white;
+}
+nav :first-child  {
```

If you want to apply the same style to different selectors you can write the selectors in one line and separation between them by `,`
also, for `li` if you want to apply a specific style by `pseudo-class` you can write `li:pseudo-class`.
See more [Pseudo-classes
](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes)
```suggestion
.nav li:first-child, .nav li:nth-child(2) {
    font-size: 20px;
    font-weight: bold;
}
```